src/app/services/search.service.ts

```diff
@@ -116,6 +116,14 @@ export class ResearchStudySearchEntry {
     }
     return '';
   }
+  /** TO-DO 
+   * Extract matchLikelihood from server response
+   * 
+   */
+  get matchLikelihood(): string | null {
+    return null;
+
```

Comment:
I think this function needs to be implemented before we can call the ticket complete. Like this, we don't actually check the score, so the display will always be null. I'm going to push a commit to implement this, and as a result of that, we'll also need to make a change on the backend to remove the score since it isn't actually being returned for base matches.